docs/gallery.rst

```diff
@@ -125,11 +122,11 @@ Clasification Visualizers
     :alt: ROC Curves for Logistic Regression
     :target: api/classifier/rocauc.html#rocauc
 
-.. image:: api/classifier/images/class_balance.png
+.. image:: api/target/images/class_balance_compare.png
     :width: 200px
     :height: 100px
-    :alt: Class Balance for Random Forest Classifier
-    :target: api/classifier/class_balance.html#class-balance
+    :alt: Class Balance
+    :target: api/target/class_balance.html#class-balance
```

Comment:
Should eventually also add the regular class balance to the gallery, but left the compare classifier since that is what was there before.